test/utils.js

```diff
@@ -107,7 +107,10 @@ export const loadSpecTestData = function (specPath, renderData) {
 
   for (const routeTemplate in spec.paths) {
     // NOTE: the template and data variable names are defined in the spec
-    const route = renderPath(routeTemplate, renderData);
+    const route = function (routeTemplate) {
```

Comment:
Aside from tests for the new sdk methods this is the only change of note.  The tests for the openapi spec were relying on the content of the examples to be synced to the tests.  with this change the tests can create tables that match whatever is in the spec example.